src/api/Metadata.js

```diff
@@ -257,6 +268,65 @@ class Metadata extends File {
         }
     }
 
+    /**
+     * Gets classification for a file.
+     *
+     * @param {BoxItem} file - File object for which we are getting classification
+     * @param {Function} successCallback - Success callback
+     * @param {Function} errorCallback - Error callback
+     * @param {boolean|void} [options.forceFetch] - Optionally Bypasses the cache
+     * @param {boolean|void} [options.refreshCache] - Optionally Updates the cache
+     * @return {Promise}
+     */
+    async getClassification(
+        file: BoxItem,
+        successCallback: Function,
+        errorCallback: Function,
+        options: FetchOptions = {},
+    ): Promise<void> {
+        const { id }: BoxItem = file;
+        if (!id) {
+            errorCallback(getBadItemError(), ERROR_CODE_FETCH_CLASSIFICATION);
+            return;
+        }
+
+        const cache: APICache = this.getCache();
+        const key = this.getClassificationCacheKey(id);
+        this.successCallback = successCallback;
+        this.errorCallback = errorCallback;
+
+        // Clear the cache if needed
+        if (options.forceFetch) {
+            cache.unset(key);
+        }
+
+        // Return the Cache value if it exists
+        if (cache.has(key)) {
+            const cachedItem = cache.get(key);
+            if (cachedItem) {
```

Comment:
null/undefined check needed?
since cache.has() checked above that there was an entry for sure.

Reply:
Great point!